Tests/Controller/Api/GalleryControllerTest.php

```diff
@@ -147,7 +147,7 @@ public function testPostGalleryMediaGalleryhasmediaAction()
         $view = $galleryController->postGalleryMediaGalleryhasmediaAction(1, 2, new Request());
 
         $this->assertInstanceOf('FOS\RestBundle\View\View', $view);
-        $this->assertSame(200, $view->getStatusCode(), 'Should return 200');
+        $this->assertSame(200, $view->getResponse()->getStatusCode(), 'Should return 200');
```

Comment:
Because the internal logic has changed. In the old version, the method `getResponse` was implicitly called.